src/cgame/cg_predict.cpp

```diff
@@ -33,10 +33,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 static  pmove_t   cg_pmove;
 
-static  int       cg_numSolidEntities;
-static  centity_t *cg_solidEntities[ MAX_ENTITIES_IN_SNAPSHOT ];
-static  int       cg_numTriggerEntities;
-static  centity_t *cg_triggerEntities[ MAX_ENTITIES_IN_SNAPSHOT ];
+static std::vector<centity_t *> cg_solidEntities;
```

Comment:
How about BoundedVector<centity_t*, MAX_GENTITIES>

Reply:
This should be fine as entities are known to be duplicated inside a snapshot. On the other hand, I'm not sure it makes sense to have a different handling here than in every other instance. Maybe a BoundedVector migration would have to start with notably `snapshot_t` and `clSnapshot_t`.

Reply:
See also the other comment about the memory usage. I'm really not sure this is a good idea.

Reply:
This is hardly a "migration" as BoundedVector does the exact same thing as the existing code here.

Reply:
Okok, so I changed it here and in `cg_buildable.cpp` for consistency. I'm not touching snapshot_t because it's in daemon, and because I'd like to avoid conflicts.